SonarQube.TeamBuild.PreProcessor/Roslyn/RoslynAnalyzerProvider.cs

```diff
@@ -63,26 +63,27 @@ public AnalyzerSettings SetupAnalyzers(ISonarQubeServer sqServer, TeamBuildSetti
                 throw new ArgumentNullException("projectKey");
             }
 
-            if (!IsCSharpPluginInstalled(sqServer))
+            AnalyzerSettings analyzerSettings = null;
+            if (IsCSharpPluginInstalled(sqServer))
             {
-                logger.LogDebug(Resources.SLAP_CSharpPluginNotInstalled);
-                return null;
-            }
+                this.server = sqServer;
+                this.settings = teamBuildSettings;
+                this.projectKey = sqProjectKey;
+                this.projectBranch = sqProjectBranch;
 
-            this.server = sqServer;
-            this.settings = teamBuildSettings;
-            this.projectKey = sqProjectKey;
-            this.projectBranch = sqProjectBranch;
-
-            AnalyzerSettings analyzerSettings = null;
+                RoslynExportProfile profile = TryGetRoslynConfigForProject();
+                if (profile != null)
+                {
+                    analyzerSettings = ProcessProfile(profile);
+                }
+                else
+                {
+                    logger.LogDebug(Resources.RAP_CSharpPluginNotInstalled);
```

Comment:
I don't have the full context here, but we should probably fail fast if the C# plugin isn't installed.